test/karma.conf.js

```diff
@@ -140,6 +141,14 @@ function getCustomLaunchers(){
       os_version: 'Yosemite'
     },
 
+    edge_bs: {
+      base: 'BrowserStack',
+      browser: 'edge',
+      browser_version: '14',
```

Comment:
You might want to leave browser version off so Browserstack can automatically use the latest Edge.